app/src/components/pages/ScheduleTimetable.tsx

```diff
@@ -56,6 +63,11 @@ function ScheduleTimetable() {
           useColorAsDefault={true}
         />
       </Scheduler>
+      <Box display="flex" justifyContent="center" marginTop="10px">

```

Comment:
I would move this above the calendar, the save button isnt visible underneath on smaller screens